core/src/main/java/com/wechat/pay/java/core/util/ObjectUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.wechat.pay.java.core.util;
+
+import com.google.gson.Gson;
+
+/** 对象操作工具类 */
+public class ObjectUtil {
+  private ObjectUtil() {}
+
+  public static final Gson gson = new Gson();
+
+  /**
+   * 返回深拷贝
+   *
+   * @param object 拷贝目标
+   * @param tClass 拷贝目标类型类
+   * @return 深拷贝
+   * @param <T> 拷贝目标对象类型
+   */
+  public static <T> T deepCopy(Object object, Class<T> tClass) {
```

Comment:
这么定义的话，可以这样调用吧，这是想要的行为吗？
```
Object1 object1 = new Object1();
Object2 copyWithWrongType = ObjectUtil.deepCopy(object1, Object2.class);
```

能不能这样？
```suggestion
  public static <T> T deepCopy(T object) {
    Class<?> tClass = object.getClass();
```

Reply:
似乎不行，因为运行时T类型会被擦除，所以无法通过`getClass`获得运行时类。所以需要提供实际的tClass